### PR TITLE
[BrokenLinksH2]

### PR DESCRIPTION
**Pull request by mohitp930**

### CODE REVIEW

**Comment by mohitp930** [comment]
Global effort to fix broken links

@mikehoffms

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. Thanks!

File: microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ self.addEventListener('notificationclick', event => {
 }, false);
 ```
 
-To learn more about notification actions, see [NotificationAction](https://developer.mozilla.org/docs/Web/API/NotificationAction) at MDN.
+To learn more about notification actions, see [NotificationAction](https://developer.mozilla.org/docs/Web/API/notification/actions) at MDN.
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges.md:195]:
```suggestion
To learn more about notification actions, see [Notification.actions](https://developer.mozilla.org/docs/Web/API/notification/actions) at MDN.
```

Comment (mikehoffms) [reply microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges.md:195]: @captainbrosset - fixes 404 at bottom of https://docs.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges#add-actions-to-notifications

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ The web platform is a collection of technologies used for building webpages, inc
 
 For functionality and compatibility reasons, Microsoft Edge adopts nearly all of the Chromium project's changes to the web platform.  Microsoft remains in full control of the Microsoft Edge browser and may defer or reject changes.  The Microsoft Edge team decides if the change benefits browser users.
 
-For information about upcoming Chromium project web platform changes, see [Chrome Platform Status Release timeline](https://www.chromestatus.com/features/schedule).
+For information about upcoming Chromium project web platform changes, see [Chrome Platform Status Release timeline](https://chromestatus.com/roadmap).
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:18]: @ericlaw1979 - fyi, url changed, fixing 404

**Pull request merged**